gapis/api/vulkan/command_splitter.go

```diff
@@ -201,24 +206,24 @@ func (t *commandSplitter) getCommandPool(ctx context.Context, id api.CmdID, queu
 		memory.Nullptr,
 		t.MustAllocWriteDataForCmd(ctx, s, t.pool).Ptr(),
 		VkResult_VK_SUCCESS,
-	), id, out)
+	), out)
 	return t.pool
 }
 
-func (t *commandSplitter) getStartedCommandBuffer(ctx context.Context, id api.CmdID, queueSubmit *VkQueueSubmit, out transform.Writer) VkCommandBuffer {
+func (t *commandSplitter) getStartedCommandBuffer(ctx context.Context, queueSubmit *VkQueueSubmit, out transform.Writer) VkCommandBuffer {
 	s := out.State()
 	a := s.Arena
 	cb := CommandBuilder{Thread: queueSubmit.Thread(), Arena: a}
 	vs := GetState(s)
 	queue := vs.Queues().Get(queueSubmit.Queue())
 
-	commandPoolID := t.getCommandPool(ctx, id, queueSubmit, out)
+	commandPoolID := t.getCommandPool(ctx, queueSubmit, out)
 
 	commandBufferAllocateInfo := NewVkCommandBufferAllocateInfo(a,
 		VkStructureType_VK_STRUCTURE_TYPE_COMMAND_BUFFER_ALLOCATE_INFO, // sType
 		NewVoidᶜᵖ(memory.Nullptr),                                      // pNext
 		commandPoolID,                                                  // commandPool
-		VkCommandBufferLevel_VK_COMMAND_BUFFER_LEVEL_PRIMARY,           // level
+		VkCommandBufferLevel_VK_COMMAND_BUFFER_LEVEL_PRIMARY, // level
```

Comment:
comment identation

Reply:
This will all get fixed up when I fix presubmit.